logrotate.d/nginx

```diff
@@ -0,0 +1,15 @@
+/var/log/nginx/*.log {
+        daily
+        missingok
+        rotate 52
```

Comment:
This will (probably) be the default in debian: I doubt this was selected by @mayli